apis/telemetry/v1alpha1/logparser_types.go

```diff
@@ -34,6 +34,7 @@ type LogParserSpec struct {
 //+kubebuilder:subresource:status
 //+kubebuilder:printcolumn:name="Status",type=string,JSONPath=`.status.conditions[-1].type`
 //+kubebuilder:printcolumn:name="Age",type=date,JSONPath=`.metadata.creationTimestamp`
+// +kubebuilder:deprecatedversion:warning="The LogParser API is deprecated, use logging in JSON format instead and leverage the JSON parsing feature of the LogPipeline"
```

Comment:
```suggestion
// +kubebuilder:deprecatedversion:warning="The LogParser API is deprecated. Instead, use the JSON format and the JSON parsing feature of the LogPipeline"
```
can we shorten this?